Sources/Core/Microsoft.StreamProcessing/Collections/FastDictionary.cs

```diff
@@ -119,7 +120,8 @@ public bool Lookup(TKey key, int hashCode, out int index)
 
                 index++;
                 if (index == this.Size) index = 0;
-            } while (true);
+            }
+            while (true);
```

Comment:
Maybe it's the ancient C++ training I had way back in the bronze age, but seeing the while in a do-while on its own line just looks confusing to me. I'll accept it if we need to for consistency with other rules, but this one in particular hurts my brain.